### ISSUE TITLE
Close Button not labeled for screen readers in information message

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2204]
**Description**
Button not labelled in information message pop up

![image(7)](https://user-images.githubusercontent.com/6835324/76282042-ced38e00-6253-11ea-925f-3d5659e1b626.png)

**Additional context**
Flagged by the a11y meetup group using Hubs
